evm/utils.js

```diff
@@ -26,6 +27,7 @@ const { CosmWasmClient } = require('@cosmjs/cosmwasm-stargate');
 const CreateDeploy = require('@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/deploy/CreateDeploy.sol/CreateDeploy.json');
 const IDeployer = require('@axelar-network/axelar-gmp-sdk-solidity/interfaces/IDeployer.json');
 const { verifyContract } = require(`${__dirname}/../axelar-chains-config`);
+const defaultGasOptions = { gasLimit: 6000000 };
```

Comment:
default can't be applicable in all cases. Let's remove it and require it be passed in from the caller if needed. We don't need to be worried about the v4.3 script